main.py

```diff
@@ -81,6 +83,28 @@ async def get_experiment_info(file: str) -> Any:
     return remote.examine(file)
 
 
+@app.get("/experiment/submit/")
+async def submit_experiment(file: str, args: str = "{}") -> int:
+    """Submit the given experiment file.
+    
+    Args:
+        file: The path of the experiment file.
+        args: The arguments to submit which must be JSONifiable to a dictionary.
+          Each key is an argument name and its value is the value of the argument.
+    
+    Returns:
+        The run identifier, an integer which is incremented at each experiment submission.
+    """
+    expid = {
+        "log_level": logging.WARNING,
+        "class_name": None,
+        "arguments": json.loads(args),
```

Comment:
I think it is fine for now, but are the scan variables JSON-native objects as well?
Maybe someday we might want to use non-JSON-native objects, just like in `qiwiscall`s.
Nevertheless, I think we don't have to cover it in this PR.

Reply:
I also agree with you.
Yet, I don't know the format of scan variables well, so I couldn't cover it in this PR.
Thus, as you said, it is okay to implement later!